index.html

```diff
@@ -1151,7 +1155,7 @@ <h3>Standard Vocabularies</h3>
 
           <dt><a href="https://w3id.org/security">Security Vocabulary</a></dt>
           <dd>
-            Definition of property names defined by the <a href="#s_di">VC Data Integrity</a> specification or the Controller Document.
+            Definition of property names defined by the <a href="#s_di">VC Data Integrity</a> specification or the Controlled Identifier Document.
```

Comment:
It seems like there must be a better way to phrase this, than `Definition of property names defined by`. Does this work?
```suggestion
            Property name definitions in the <a href="#s_di">VC Data Integrity</a> specification or the Controlled Identifier Document.
```